src/analytics/service/AnalyticsService.ts

```diff
@@ -0,0 +1,53 @@
+import { AxiosResponse, AxiosRequestConfig } from 'axios';
+import * as Q from 'q';
+
+import Resource from '../../model/Resource';
+import Version from '../../model/Version';
+import RequestClientWrapper from '../../RequestClientWrapper';
+import GenerateApiError from '../../utils/GenerateApiError';
+import Embed from '../model/Embed';
+
+const EXPECTED_POST_STATUS = 200;

```

Comment:
@ryankazokas I must have missed this on the API review but why are you returning a 200 here instead of a 201?

Since we add the current time into generation of the embed URL, this POST is not idempotent.

Please update the API to return a 201 for this request then change the expected POST status here.